lib/node_modules/@stdlib/repl/README.md

```diff
@@ -83,6 +84,28 @@ The function supports specifying the following settings:
 -   **autoDeletePairs**: boolean indicating whether to automatically delete adjacent matching brackets, parentheses, and quotes. Default: `true`.
 -   **autoPage**: boolean indicating whether to automatically page return values having a display size exceeding the visible screen. When streams are TTY, the default is `true`; otherwise, the default is `false`.
 -   **completionPreviews**: boolean indicating whether to display completion previews for auto-completion. When streams are TTY, the default is `true`; otherwise, the default is `false`.
+-   **syntaxHighlighting**: boolean indicating whether to enable syntax highlighting of the input. When streams are TTY, the default is `true`; otherwise, the default is `false`.
+-   **theme**: initial color theme for syntax highlighting. Default: `default`.
```

Comment:
@Snehil-Shah I thought the plan was to rename to something other than "default"?

Reply:
@kgryte I'll do that in the next PR where I'll add more built-in themes

Reply:
Not clear why we should wait. And as a matter of principle, changing the name from 'default' to something else in a future PR is effectively a "breaking change", as user code previously using "default" as the name would no longer work. As such, I think it makes more sense to rename in this PR and it should be a trivial amount of work to do so.